sdk/core/amqp-common/changelog.md

```diff
@@ -1,3 +1,7 @@
+### 2019-5-6 1.0.0-preview.4
+
+- Moved repo from https://github.com/Azure/amqp-common-js to https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/amqp-common [Issue 1208](https://github.com/Azure/azure-sdk-for-js/issues/1208)
```

Comment:
@ramya0820 
Can you update that link to point to the release tag instead so that it will be permanent? https://github.com/Azure/azure-sdk-for-js/tree/%40azure/amqp-common_1.0.0-preview.4/sdk/core/amqp-common